cmd/obslytics/export.go

```diff
@@ -2,41 +2,110 @@ package main
 
 import (
 	"context"
+	"fmt"
+	"os"
+	"time"
 
 	"github.com/go-kit/kit/log"
 	"github.com/oklog/run"
-	infactory "github.com/thanos-community/obslytics/pkg/input/factory"
-	outfactory "github.com/thanos-community/obslytics/pkg/output/factory"
+	"github.com/thanos-community/obslytics/pkg/input"
+	"github.com/thanos-community/obslytics/pkg/output"
+
+	"github.com/thanos-community/obslytics/pkg/output/debug"
 	"github.com/thanos-io/thanos/pkg/extflag"
 	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/thanos-community/obslytics/pkg/ingest"
+	infactory "github.com/thanos-community/obslytics/pkg/input/factory"
+	outfactory "github.com/thanos-community/obslytics/pkg/output/factory"
 )
 
 func registerExport(m map[string]setupFunc, app *kingpin.Application) {
 	cmd := app.Command("export", "Export Observability Data into popular analytics formats.")
-	inputFlag := extflag.RegisterPathOrContent(cmd, "input", "YAML for input configuration.", true)
-	outputFlag := extflag.RegisterPathOrContent(cmd, "ouput", "YAML for output configuration.", true)
+	inputFlag := extflag.RegisterPathOrContent(cmd, "input-cfg", "YAML for input configuration.", true)
+	// TODO(inecas): add support for output configuration
+	outputFlag := extflag.RegisterPathOrContent(cmd, "ouput-cfg", "YAML for output configuration.", false)
+
+	serParams := input.SeriesParams{}
+	cmd.Flag("metric", "Name of the metric to export").Required().StringVar(&serParams.Metric)
+	timeFmt := time.RFC3339
+
+	minTimeStr := cmd.Flag("min-time", fmt.Sprintf("The lower boundary of the time series in %s format", timeFmt)).
```

Comment:
What about using nice flag we have in Thanos: https://github.com/thanos-io/thanos/blob/2ea2c2b7c4e6ac157925a3a9590dd5ac47bb1ca9/cmd/thanos/sidecar.go#L428

timeorduration

Reply:
Great suggestion: I've tried to look at some patterns around that in prometheus codebase, but haven't. Allowed getting rid of some unnecessary lines.